docker/Dockerfile.build

```diff
@@ -0,0 +1,74 @@
+# .deb building Dockerfile
+# This Dockerfile will build an image that can be used to create a debian package from latest code in the develop branch. The completed debian package will need to be copied off the image after creation (see below invocation)
+
+# Build the image for creating the package
+# $ docker build --file docker/Dockerfile.build --tag "portal_builddeb:test" .
+
+# Build the image
+# $ docker run "portal_builddeb:test"
+
+# Build the image (override defaults)
+# $ docker run --tty -e REPO_SLUG='ivan-c/true_nth_usa_portal' -e BRANCH='feature/deb-packaging_dockerize-portal_compose' "portal_builddeb:test"
```

Comment:
I wanted a concrete example of how to override the [default](https://github.com/uwcirg/true_nth_usa_portal/pull/414/files/02331b6bcf3fd199684cda63c5baa6bd3a3daafb#diff-1633c0903adc235424069b03f49ac523R66) of `uwcirg/true_nth_usa_portal` set in the Dockerfile. I'll make it a little less specific